src/generator/writers/writeAccessors.ts

```diff
@@ -55,10 +69,30 @@ export function writeAccessors(
             if (flatPack.length !== flatTypes.length)
                 throw Error("Flat pack and flat types length mismatch");
             const pairs = flatPack.map((v, i) => `${flatTypes[i]} ${v}`);
-            ctx.used(`__tact_tuple_destroy_${flatPack.length}`);
-            ctx.append(
-                `var (${pairs.join(", ")}) = __tact_tuple_destroy_${flatPack.length}(v);`,
-            );
+            if (flatPack.length < 16) {
```

Comment:
we should replace all these magic 14, 15, 16 numbers with either constants or some expressions derived from constants that convey the semantics in a clearer way